### PR TITLE
modify the position of company and business dropdown

### PR DESCRIPTION
**Pull request by ian-bpoc**
# Pull Request Review Comments

## Overview
- **Purpose of the PR**:
    - [/ ] Bug fix
    - [/ ] Refactoring

## Specific Areas for Verification
- Please specify any particular requests or questions for the reviewer here.

## Suggestions/Concerns
- If you have any suggestions or concerns, please mention them here.

---

_**Additional Comments for the Reviewer (Optional)**_

- Add any additional comments or information for the reviewer here.

### CODE REVIEW

File: backend/api/urls.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
     path("master-business-division/bulk-update/", views.MasterBusinessDivisionUpdate.as_view(), name="master-business-division-update"),
     path("master-business-division/<str:pk>/delete/", views.MasterBusinessDivisionDestroy.as_view(), name="master-business-division-delete"),
     
+    path("business-divisions/", views.CompaniesWithBusinessDivisions.as_view(), name="companies-with-business-division"), #url for filtering the bussiness division from the selected company
```
Comment (edward-stalley-work) [inline backend/api/urls.py:11]: Spelling - please change to this: (remove extra 's')

#url for filtering the business division from the selected company

File: client/src/pages/Employees/EmployeesRegistration.tsx [hunk 2]
```diff
@@ -24,6 +24,8 @@ const EmployeesRegistration = () => {
     const dispatch = useDispatch()
     const [businessSelection, setBusinessSelection] = useState<any>([])
     const [companySelection, setCompanySelection] = useState<any>([])
+    const [selectedCompanyId, setSelectedCompanyId] = useState('');
+
```
Comment (edward-stalley-work) [inline client/src/pages/Employees/EmployeesRegistration.tsx:28]: can remove this space

File: client/src/pages/Employees/EmployeesRegistration.tsx [hunk 3]
```diff
@@ -39,8 +41,8 @@ const EmployeesRegistration = () => {
 
     const fetchData = async () => {
       try {
-        const resBusinessDivisions = await dispatch(fetchBusinessDivisions() as unknown as UnknownAction)
-        setBusinessSelection(resBusinessDivisions.payload)
+        // const resBusinessDivisions = await dispatch(fetchBusinessDivisions() as unknown as UnknownAction)
+        // setBusinessSelection(resBusinessDivisions.payload)
```
Comment (edward-stalley-work) [inline client/src/pages/Employees/EmployeesRegistration.tsx:45]: do we need this comment code?
if not please remove!

File: client/src/pages/Employees/EmployeesRegistration.tsx [hunk 4]
```diff
@@ -24,6 +24,8 @@ const EmployeesRegistration = () => {
     const dispatch = useDispatch()
     const [businessSelection, setBusinessSelection] = useState<any>([])
```
Comment (edward-stalley-work) [inline client/src/pages/Employees/EmployeesRegistration.tsx:25]: could this and all related code be renamed to businessDivisionSelection?

**Review by edward-stalley-work — COMMENTED** [review]
Looks goo Ian thank you.
Just a few small changes.

Would it also be possible to do the same implementation for the edit screen.
So you can only select business divisions based on the company?
In list column can we change the column header order.
Eg.   `|company name| business division| created by`

**Comment by ian-bpoc** [comment]
> Looks good Ian - thank you. Just a few small changes.
> 
> Would it also be possible to do the same implementation for the edit screen. So you can only select business divisions based on the company? In list column can we change the column header order. Eg. `|company name| business division| created by`

maybe i can suggest Ed to create new ticket for the screen, in order also to avoid more conflicts with my branch, since it might be behind from the develop

File: backend/api/urls.py (hunk 1 above)
Comment (ian-bpoc) [reply backend/api/urls.py]: fixed

File: client/src/pages/Employees/EmployeesRegistration.tsx (hunk 2 above)
Comment (ian-bpoc) [reply client/src/pages/Employees/EmployeesRegistration.tsx]: fixed

File: client/src/pages/Employees/EmployeesRegistration.tsx (hunk 3 above)
Comment (ian-bpoc) [reply client/src/pages/Employees/EmployeesRegistration.tsx]: fixed

File: client/src/pages/Employees/EmployeesRegistration.tsx (hunk 4 above)
Comment (ian-bpoc) [reply client/src/pages/Employees/EmployeesRegistration.tsx]: fixed

File: backend/api/urls.py [hunk 5]
```diff
@@ -7,8 +7,7 @@
     path("master-business-division/<int:pk>", views.BusinessDivisionMasterRetrieve.as_view(), name="master-business-division-detail"),
     path("master-business-division/bulk-update/", views.MasterBusinessDivisionUpdate.as_view(), name="master-business-division-update"),
     path("master-business-division/<str:pk>/delete/", views.MasterBusinessDivisionDestroy.as_view(), name="master-business-division-delete"),
-    
-    path("business-divisions/", views.CompaniesWithBusinessDivisions.as_view(), name="companies-with-business-division"), #url for filtering the bussiness division from the selected company
+    path("business-division/", views.CompaniesWithBusinessDivisions.as_view(), name="companies-with-business-division"),
```
Comment (edward-stalley-work) [inline backend/api/urls.py:10]: The comment was fine - I think it was helpful actually.
Please put the comment back with the spelling mistake fixed.

bussiness → business

**Review by edward-stalley-work — CHANGES_REQUESTED** [review]
One small edit thank you Ian

File: backend/api/urls.py (hunk 5 above)
Comment (edward-stalley-work) [reply backend/api/urls.py]: fixed

**Pull request merged**